sql/core/src/test/scala/org/apache/spark/sql/JoinHintSuite.scala

```diff
@@ -622,28 +632,23 @@ class JoinHintSuite extends PlanTest with SharedSparkSession with AdaptiveSparkP
         withLogAppender(hintAppender, level = Some(Level.WARN)) {
           assertBroadcastHashJoin(
             df1.join(df2.hint("BROADCAST"), $"a1" === $"b1", joinType), BuildRight)
-          assertShuffleHashJoin(
-            df1.join(df2.hint("SHUFFLE_HASH"), $"a1" === $"b1", joinType), BuildRight)
         }
 
         val logs = hintAppender.loggingEvents.map(_.getMessage.getFormattedMessage)
           .filter(_.contains("is not supported in the query:"))
         assert(logs.isEmpty)
       }
 
-      Seq("right_outer").foreach { joinType =>
+      Seq("left_semi", "left_anti").foreach { joinType =>
         val hintAppender = new LogAppender(s"join hint build side check for $joinType")
         withLogAppender(hintAppender, level = Some(Level.WARN)) {
-          assertShuffleMergeJoin(
-            df1.join(df2.hint("BROADCAST"), $"a1" === $"b1", joinType))
-          assertShuffleMergeJoin(
-            df1.join(df2.hint("SHUFFLE_HASH"), $"a1" === $"b1", joinType))
```

Comment:
Shall we still keep the `hint("BROADCAST")` test and change the `hint("SHUFFLE_HASH")` to `assertShuffleHashJoin`?

Reply:
I think I mistyped after a long day, and accidentally got rid of the right_outer test, I reverted it.  

But I took a look at the SPARK issue (SPARK-35221) and as it is testing whether there are warning messages, if we change to assertShuffleHashJoin, the warming msg check further down will not pass.  Let me know what you think.

Reply:
I see. Then no need to keep this. Thanks for checking the original issue!